wolfhsm/wh_server_dma.h

```diff
@@ -0,0 +1,107 @@
+#ifndef WH_SERVER_DMA_H_
+#define WH_SERVER_DMA_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "wolfhsm/wh_server.h"
+
+#define WH_DMA_ADDR_ALLOWLIST_SIZE (10)
```

Comment:
Consider using NUM/COUNT instead of SIZE.  SIZE/LEN makes me think its the number of bytes, not elements.

Reply:
fixed